_episodes/03-computing-pi.md

```diff
@@ -61,10 +59,15 @@ the blue circle M compared to the green square N. Then π is approximated by the
 > >             M += 1
 > >     return 4 * M / N
 > >
-> > %timeit calc_pi(10**6)
-> >
+> > %timeit calc_pi(1e6)
```

Comment:
This should be `10**6` because we need integers

Reply:
Sorry this is really stupid.